docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.8"
+services:
+    ngrok:
+        image: wernight/ngrok:latest
+        ports:
+        - 4040:4040
+        environment:
+            NGROK_PROTOCOL: http
+            NGROK_PORT: atlantis:4141
+        depends_on:
+        - atlantis
+    atlantis:
+        build:
+            context: .
+            dockerfile: Dockerfile.dev
+        ports:
+        - 4141:4141
+        volumes:
+            - ~/.ssh:/.ssh
+            - ~/go/src/github.com/runatlantis/atlantis:/go/src/github.com/runatlantis/atlantis
```

Comment:
I don't think it makes sense to hardcode this, GOPATH isn't required anymore. Can you use `./`?

Reply:
Gotcha, can change it to something more generic.